pkg/xds/cache/cla/cache.go

```diff
@@ -69,11 +64,12 @@ func (c *Cache) GetCLA(ctx context.Context, meshName, meshHash, service string,
 		if err := c.rm.Get(ctx, mesh, core_store.GetByKey(meshName, model.NoMesh)); err != nil {
 			return nil, err
 		}
-		externalServices := &core_mesh.ExternalServiceResourceList{}
-		if err := c.rm.List(ctx, externalServices, core_store.ListByMesh(meshName)); err != nil {
-			return nil, err
-		}
-		endpointMap := topology.BuildEndpointMap(mesh, c.zone, dataplanes.Items, externalServices.Items, c.dsl)
+		// External Services can be nil since GetCLA is used only for EDS clusters
```

Comment:
Do you think we still need this comment? There are no references to external services in the code anymore

Reply:
true